spec/rubocop/cop/rspec/identical_equality_assertion_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IdenticalEqualityAssertion do
+  let(:msg) do
+    'Identical expressions on both sides of the equality '\
+    'may indicate a flawed test.'
+  end
+
+  it 'registers an offense when using identical expressions with `eq`' do
+    expect_offense(<<~RUBY)
+      expect(foo.bar).to eq(foo.bar)
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ #{msg}
```

Comment:
I prefer not having the message interpolated in the specs – I also don't think we do it in other specs.

What do you think about e.g.
```suggestion
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Identical expressions on both sides of the equality may indicate a flawed test.
```
?

Reply:
Changed.